### PR TITLE
Support Scala 2.13

### PR DESCRIPTION
**Pull request by marcus-nl**
Also upgraded the dependencies, since these later versions have 2.13 releases.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,15 +4,14 @@ organization := "com.github.andyglow"
...
-crossScalaVersions := Seq("2.11.12", "2.12.4")
+crossScalaVersions := Seq("2.11.12", "2.12.4", "2.13.0")
 
 scalacOptions ++= Seq(
   "-encoding", "UTF-8",
   "-feature",
   "-unchecked",
   "-deprecation",
   "-Xlint",
-  "-Yno-adapted-args",
```
Comment (marcus-nl) [inline build.sbt:15]: This option is no longer supported by Scala 2.13, but -Xlint enables the warning.

Comment (andyglow) [reply build.sbt:15]: oh. let me look at this. there was a way to separate options needed for one scala version from another.

Comment (andyglow) [reply build.sbt:15]: i have just published `2.0.4` version with scala 2.13 support. will be available on maven soon. thanks for the contribution!

Comment (marcus-nl) [reply build.sbt:15]: Great thanks!

**Pull request merged**